src/components/Popover/PopoverTrigger.tsx

```diff
@@ -28,6 +28,9 @@ const PopoverTrigger = React.forwardRef<
       {...rest}
     />
   );
-}) as Polymorphic.ForwardRefComponent<'button', PopoverTriggerProps & { as: React.ElementType }>;
+}) as Polymorphic.ForwardRefComponent<
+  React.ElementType,
```

Comment:
shouldn't be `ElementType`, needs to be `button` here . Why are we making this change?

Reply:
Ok  then :/